### DISCUSSION TITLE
breaking changes after beta? and release date of V2.0.0?

### DISCUSSION THREAD

**Discussion by arashi-dev** [tauri-apps/tauri#9010] [Q&A] [answered]
When is the stable version of V2 expected to be released?
And will there be any API changes after the beta release?

**Comment by FabianLars** [comment, marked as answer]
No breaking changes are expected once we reach release candidate (or stable afterwards). The beta might still get a few breaking changes now that we get more feedback and of course the ongoing audit may require some too.

The release candidate is planned for late Q1/early Q2, stable shortly afterwards.

**Reply by arashi-dev** [reply]
Thank you very much. Good luck :)
